action/Run-Maester.ps1

```diff
@@ -47,7 +47,7 @@ BEGIN {
     Write-Host 'Starting Maester tests'
 }
 PROCESS {
-    $graphToken = Get-AzAccessToken -ResourceUrl 'https://graph.microsoft.com' -AsSecureString
+    $graphToken = Get-MtAccessTokenUsingCli -ResourceUrl 'https://graph.microsoft.com' -AsSecureString
```

Comment:
Can you please update this to support the use of different environments (worldwide, GCC, GCC-High, etc)? I know that's not as simple of a change, and the current implementation has the global endpoint URL hard coded, so feel free to defer that improvement for a later PR.

Reply:
Are the application IDs or resources known for those requests?
As in all regions and then these resources. Graph, Teams, Exchange.

I recon we can make it accept a "cloud/region" and then use a table to get the correct tokens. If Azure cli supports that off-course

Reply:
I _think_ you would just need to account for the different API endpoints for each sovereign environment.
https://learn.microsoft.com/en-us/entra/identity-platform/authentication-national-cloud